language/python/build-images.md

```diff
@@ -25,10 +25,19 @@ To complete this tutorial, you need the following:
 Let’s create a simple Python application using the Flask framework that we’ll use as our example. Create a directory in your local machine named `python-docker` and follow the steps below to create a simple web server.
 
 ```shell
-$ cd /path/to/python-docker
-$ pip3 install Flask
-$ pip3 freeze > requirements.txt
-$ touch app.py
+cd /path/to/python-docker
+pip3 install Flask
+pip3 freeze > requirements.txt
+touch app.py
```

Comment:
I suggest to remove this `touch app.py` line. Then we won't need the PowerShell section.

Reply:
In all other places in our docs we show commands normally with a shell prompt `$`. I know in PowerShell it looks different, but we prefer to keep it to be consistent to the rest of the docs.

Reply:
The `touch` command doesn't work for windows anyway - as I told here: #13124 
Maybe it's better to make that clear and to tell people to create a file in a different way on windows?